installation-bundle/src/Database/Installer.php

```diff
@@ -54,22 +59,22 @@ public function getCommands(): array
 
     /**
      * @throws \InvalidArgumentException
+     * @throws \Doctrine\DBAL\DBALException
      */
-    public function execCommand(string $hash): void
+    public function execCommands(array $hashes): void
     {
         if (null === $this->commands) {
             $this->compileCommands();
         }
 
-        foreach ($this->commands as $commands) {
-            if (isset($commands[$hash])) {
-                $this->connection->query($commands[$hash]);
+        $commands = [];
+        array_map(function ($c) use (&$commands): void {
```

Comment:
What exactly does this do?

Reply:
This just flattens the ``$commands`` array from something like this

```PHP
[
    'CREATE' => [
        'hash1' => 'sql1',
        'hash2' => 'sql2'
    ],
    'ALTER_TABLE' => [
        'hash3' => 'sql3'
    ],
    //...
];
```
to something like this:

```PHP
[
    'hash1' => 'sql1',
    'hash2' => 'sql2'
    'hash3' => 'sql3'
    //...
];
```

Reply:
These four lines can probably be simplified to:

```php
$commands = array_reduce($this->commands, 'array_merge', []);
```

WDYT?